streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorStateManagerTest.java

```diff
@@ -210,30 +207,6 @@ public void shouldFindSingleStoreForChangelog() {
         );
     }
 
-    @Test
-    public void shouldRestoreStoreWithRestoreCallback() {
```

Comment:
It doesn't look like this was related to the bulk loading, but I'm guessing it was.

Reply:
Ah good catch, I should not remove this test; instead I'd just replace the removed MockBatchingStateRestoreListener with MockStateStoreCallback.

Reply:
Thanks!